brands/shop/bakery.json

```diff
@@ -82,6 +82,17 @@
       "shop": "bakery"
     }
   },
+  "shop/bakery|Breadtop": {
+    "countryCodes": ["au"],
+    "tags": {
+      "brand": "Breadtop",
+      "brand:wikidata": "Q4959217",
+      "brand:wikipedia": "en:Breadtop",
+      "cuisine": "chinese",
```

Comment:
I am not an expert but from looking at their website they use asian-related styling but products seems to not be limited to a Chinese cuisine.

Wikipedia article describes it as "selling a wide range of bread, cakes, buns and pastries".

Are you sure that `cuisine=chinese` tag fits here? (again, just checking is it intentional)

Reply:
Yes the products themselves are mostly Chinese cuisine (with the exception of the odd croissant), the cuisine they serve is different to an European or typical Australian (which would mostly be european) bakery.